tools/setup_helpers/extension.py

```diff
@@ -72,6 +76,7 @@ def build_extension(self, ext):
             "-DBUILD_SHARED_LIBS=OFF",
             "-DCMAKE_POLICY_DEFAULT_CMP0063=NEW",
             "-DSPM_ENABLE_SHARED=OFF",
+            f"-DCMAKE_CXX_FLAGS={_get_cxx11_abi()}",
```

Comment:
I am not sure if `-DCMAKE_CXX_FLAGS` appends the provided value to default values or overwrites them. Can you do the research to make sure it's file, or perhaps use custom option name to avoid potential conflict?

Reply:
Added a cache variable to store this value. Lmk if this looks okay to you.